src/contentChunk.js

```diff
@@ -61,3 +61,29 @@ exports.extractStringPayloadIntoChunks = (payload) => {
     stream.end(payload)
   })
 }
+
+export const retrievePayload = (fileId, callback) => {
```

Comment:
To keep these functions consistent, can we update this method to be an async function instead of having a callback